types/index.ts

```diff
@@ -1 +1,8 @@
 export * from './resource';
+export * from './author';
+export * from './blockchain';
+export * from './glossary';
+export * from './category';
+export * from './contributor';
+export * from './tag';
+
```

Comment:
I did a brief search for this but it looked like all of the answers required a babel plugin of some sort.

Is there not a better way to do this? I felt crazy writing this out.

Reply:
Nope, this is the expected pattern. Honestly at this point I'd say it's easier to just merge them all into a single `types.ts` file until we find that's too complex.